app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -151,4 +152,26 @@ def textual_containers
     end
     h
   end
+
+  def textual_container_image_registries
```

Comment:
Same for these ones (need simpler re-implementation in `container_summary_helper.rb`).